WinCompatibilityPack/WinCompatibilityPack.psd1

```diff
@@ -135,6 +136,12 @@ This is the first release of this module with the basic commands:
 These commands provide a set of tools allowing you to run Windows PowerShell
 commands from PowerShell Core (PowerShell 6). See the help for the
 indivdual commands for examples on how to use this functionality.
+
+Additionall, the command `Add-WindowsPSModulePath` enables enumerating
```

Comment:
`Additionall` -> `Additionally`

Also, while you are in here, can you fix `indivdual` - `individual` on line 138?